### PR TITLE
Implemented Agencies module

### PR DESCRIPTION
**Pull request by Mloweedgar**

### CODE REVIEW

**Review by BenMaruchu — CHANGES_REQUESTED** [review]
Can you use the added organization icon? It is already in develop branch

**Comment by Mloweedgar** [comment]
I have updated the organisation icon already @BenMaruchu

**Comment by navish** [comment]
@Mloweedgar Can you update the name in Agencies.js from StakeholdersContactsLayout to StakeholdersAgenciesLayout

**Comment by Mloweedgar** [comment]
I think may be you should first look at how other layout files are named first, like Contacts.js, Notifications.js and Roles.js @navish

**Comment by BenMaruchu** [comment]
@Mloweedgar pull from develop ... there is a conflict resolve it the push

**Comment by Mloweedgar** [comment]
@BenMaruchu the conflict has been resolved  already

File: src/layouts/BaseLayout.js [hunk 1]
```diff
@@ -201,8 +201,10 @@ const breadcrumbNameMap = {
     title: 'List of available warehouses',
   },
   /* Stakeholders Routes */
```
Comment (BenMaruchu) [inline src/layouts/BaseLayout.js]: So the navigation to agencies won't be available here

**Review by BenMaruchu — CHANGES_REQUESTED** [review]

**Comment by navish** [comment]
> I think may be you should first look at how other layout files are named first, like Contacts.js, Notifications.js and Roles.js @navish

I have.

File: src/Stakeholders/layouts/Agencies.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+// import UIState from '../../components/UIState';
+import Agencies from '../components/Agencies';
+/**
+ * @function
+ * @name StakeholdersContactsLayout
```
Comment (navish) [inline src/Stakeholders/layouts/Agencies.js]: Update the name from StakeholdersContactsLayout to StakeholdersAgenciesLayout

**Review by navish — CHANGES_REQUESTED** [review]

File: src/layouts/BaseLayout.js (hunk 1 above)
Comment (Mloweedgar) [reply src/layouts/BaseLayout.js]: Okay, I have fixed it

File: src/Stakeholders/layouts/Agencies.js (hunk 2 above)
Comment (Mloweedgar) [reply src/Stakeholders/layouts/Agencies.js]: Okay, I have fixed it

**Review by BenMaruchu — APPROVED** [review]

**Review by navish — APPROVED** [review]

**Pull request merged**